### PR TITLE
Make NPM installable

### PR DESCRIPTION
**Pull request by taylormck**
## Issue
#8

## Solution
I just added a `package.json` and updated the install instructions.
I haven't published this yet; figured that should be done from master once this lands.

I used `Bazaarvoice` as the author and `BV Apache 2.0 License` as the license.
If anyone thinks it should be something else, let me know.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+    "url": "git://github.com/bazaarvoice/git-land.git"
+  },
+  "keywords": [
+    "git",
+    "cli"
+  ],
+  "bin": {
+    "git-land": "git-land"
+  },
+  "author": "Bazaarvoice",
```
Comment (lawnsea) [inline package.json]: Put me (lon.ingram@bazaarvoice.com) as the author and add a contributors section, please

File: README.md [hunk 2]
```diff
@@ -40,8 +40,7 @@ Put the bash script in a folder that is in your `PATH` and make it executable.
 For example, to install it to `~/bin/`, do the following:
 
 ```sh
-curl -o ~/bin/git-land https://raw.githubusercontent.com/bazaarvoice/git-land/master/git-land
-chmod +x ~/bin/git-land
+npm install --global git-land
```
Comment (lawnsea) [inline README.md]: Instead of replacing the previous installation instructions, append instructions for installing via npm.

File: package.json [hunk 3]
```diff
@@ -0,0 +1,23 @@
...
+  "license": "BV Apache 2.0 License",
```
Comment (lawnsea) [inline package.json]: According to https://docs.npmjs.com/files/package.json#license, the right thing for us to do is `"license" : "LicenseRef-LICENSE"` and rename `LICENSE.md` to `LICENSE`.

File: package.json [hunk 4]
```diff
@@ -0,0 +1,23 @@
+{
+  "name": "git-land",
+  "version": "0.0.0",
```
Comment (lawnsea) [inline package.json]: Let's call this version 1.0.0, please.

**Comment by lawnsea** [comment]
Other than the comments above, this looks great, @taylormck. Thanks!

**Comment by brianarn** [comment]
@lawnsea Looks like feedback has been addressed, would you be able to re-review and land?

**Comment by lawnsea** [comment]
Sure. I was waiting for a go-ahead from @taylormck.

**Comment by lawnsea** [comment]
:+1:

**Pull request closed without merging**